src/client/context.ts

```diff
@@ -35,22 +36,60 @@ export interface Context {
     id?: {[key: string]: string|undefined;};
 
     /**
+     * @hidden
      * A context object is open for extension with any custom properties/metadata.
      */
     [key: string]: unknown;
 }
 
-// Built-in contexts
+/**
+ * Built-in context to define a contact.
+ */
 export interface ContactContext extends Context {
+    /**
+     * The context type is always 'fdc3.contact'.
+     */
     type: 'fdc3.contact';
+    /**
+     * Name of the contact.
+     */
     name: string;
+    /**
+     * The contact data. Currently has optional fields for email address, Twitter handle, and phone number.
+     */
     id: {[key: string]: string}&{email?: string; twitter?: string; phone?: string};
 }
-export interface SecurityContext extends Context {
-    type: 'fdc3.security';
-    id: {[key: string]: string}&{default: string};
+
+/**
+ * Built-in context to define a financial instrument.
+ */
+export interface InstrumentContext extends Context {
```

Comment:
I think this is correct, because we require the well-understood id fields to be filled in. The name isn't that much use.

Reply:
What I'm saying is, can we make the doc comment for name reflect the meaning of name for this specific derived type instead of just having it re-print the base type's comment, or if name is not a valid property for an InstrumentContext can we make it clear that it should be undefined for the derived type.

Reply:
Aah! Yes that makes sense, fixing